setup.py

```diff
@@ -197,7 +197,8 @@ def compile_KLU():
         "scikit-fem>=0.2.0",
         "casadi>=3.5.0",
         "imageio>=2.9.0",
-        "jupyter",  # For example notebooks
+        "importlib-metadata",
+        "nbconvert",
```

Comment:
IMO `nbconvert` should also be a `dev` dependency, given that it is only used for testing.

Reply:
Before making `jupyter` a `dev` dependency `nbconvert` was being installed with that only but after moving `jupyter` to dev dependency it was giving same error as `importlib-metadata` so I had to explicitly list it as required dependency.
That being said, i may try moving it to dev dependency as well if you suggest. I'll wait for your response to do that after this comment.

Reply:
Yes, `nbconvert` should be made a `dev` dependency.

Reply:
Did it, therefore marking it as resolved.